mne/encoding/feature.py

```diff
@@ -0,0 +1,269 @@
+"""Classes for transforming, preprocessing, and fitting encoding models."""
+# Authors: Chris Holdgraf <choldgraf@gmail.com>
+#          Jona Sassenhagen <jona.sassenhagen@gmail.de>
+#          Jean-Remi King <jeanremi.king@gmail.com>
+#
+# License: BSD (3-clause)
+import numpy as np
+from scipy.sparse import spmatrix, csr_matrix
+from ..utils import warn
+
+
+class FeatureDelayer(object):
```

Comment:
Hankel would match, but does not speak much to most of us. 
something with 'time lag' might be more explicit.

FYI, I called it `HankelCovariances` in pyRiemann when it include the covariance estimation, and i was about to push a new transformer with the ugly name of `TimeDelayer` :(